force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -139,15 +162,14 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String returnHelloWorld() {
         // Initialize the result String
+        String stringResult = '';
 
         // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
+         for (Integer i = 0; i < 3; i++){
+            stringResult += 'Hello World!; ';
          }
-        */
 
-        return null; // Replace null with the variable you used to store the result
+        return stringResult;
```

Comment:
Eventually we will learn easier/shortcut ways of doing this. String.repeat() for example!